plugins/modules/net_tools/nmcli.py

```diff
@@ -1288,6 +1341,26 @@
     autoconnect: true
     state: present
 
+- name: Create a VPN L2TP connection for ansible_user to connect on vpn.example.com authenticating with user 'brittany' and pre-shared key as 'Brittany123'
```

Comment:
This line is pretty long and will require horizontal scrolling in the docs for many devices, better to shorten it:
```suggestion
- name: >-
    Create a VPN L2TP connection for ansible_user to connect on vpn.example.com
    authenticating with user 'brittany' and pre-shared key as 'Brittany123'
```